src/utilities/extendSchema.js

```diff
@@ -439,9 +439,9 @@ export function extendSchema(
       description: type.description,
       astNode: type.astNode,
       extensionASTNodes,
-      serialize: type._scalarConfig.serialize,
-      parseValue: type._scalarConfig.parseValue,
-      parseLiteral: type._scalarConfig.parseLiteral,
```

Comment:
At the moment to copy scalar config you should either use `type._scalarConfig.serialize` or `type.serialize.bind(type)`.
This is a big problem for 3rd-party libraries doing schema transformations.